File: ucrop/src/main/java/com/yalantis/ucrop/callback/BitmapLoadCallback.java
package com.yalantis.ucrop.callback;

import android.graphics.Bitmap;

import com.yalantis.ucrop.model.ExifInfo;

public interface BitmapLoadCallback {

    void onBitmapLoaded(Bitmap bitmap,ExifInfo exifInfo,String imageInputPath,String imageOutputPath);

    void onFailure(Exception bitmapWorkerException);

}

File: app/src/main/java/me/duckfollow/ozone/view/crop/callback/BitmapLoadCallback.kt
package me.duckfollow.ozone.view.crop.callback

import android.graphics.Bitmap
import me.duckfollow.ozone.view.crop.model.ExifInfo

interface BitmapLoadCallback {

    fun onBitmapLoaded(
        bitmap: Bitmap,
        exifInfo: ExifInfo,
        imageInputPath: String,
        imageOutputPath: String?
    )

    fun onFailure(bitmapWorkerException: Exception)

}